glitch/apps.py

```diff
@@ -15,31 +16,59 @@
     read_frame,
 )
 
+block_aspects = [
```

Comment:
I like this of having predefined aspect ratios! later we can think how to incorporate weirder shapes 😄 
constants names should be in all caps.